redesign/js/app/FileForm.js

```diff
@@ -174,8 +174,26 @@ $.extend( FileForm.prototype, {
 	 * @param {ApplicationError} error
 	 */
 	_displayError: function( error ) {
-		// FIXME: Use the bootstrap error thing.
 		console.log( error.getMessage() );
+
+		var $fileFormAlert = $( '#file-form-alert' );
+		var $fileFormAlertPlaceholder = document.getElementById( 'file-form-alert-placeholder' );
+		while( $fileFormAlertPlaceholder.firstChild ) {
+			$fileFormAlertPlaceholder.removeChild( $fileFormAlertPlaceholder.firstChild );
+		}
+		$fileFormAlertPlaceholder.appendChild( document.createTextNode( error.getMessage() ) );
+
+		// TODO animation slide in
+		$fileFormAlert.show();
+	},
+
+	/**
+	 * Displays an error on the front-page.
+	 */
+	_dismissError: function() {
+		var $fileFormAlert = $( '#file-form-alert' );
+		// TODO animation slide out
+		$fileFormAlert.hide();
```

Comment:
I think this should just be:

	_displayError: function( error ) {
		$( '#file-form-alert' ).slideDown();
		$( '#file-form-alert-placeholder' ).text( error.getMessage() );
	},

and

	_dismissError: function() {
		$( '#file-form-alert' ).slideUp();
	},